lib/utils/lifecycle.js

```diff
@@ -262,6 +262,28 @@ function runCmd_ (cmd, pkg, env, wd, stage, unsafe, uid, gid, cb_) {
     sh = process.env.comspec || 'cmd'
     shFlag = '/d /s /c'
     conf.windowsVerbatimArguments = true
+
+    // check if we have a composed command
+    // e.g.: `"test": "node ../test/test"`
+    if (cmd.indexOf(' ') !== -1) {
+      // split the command
+      var cmdChunks = cmd.split(' ')
```

Comment:
Are you sure you want to do a `split()` like that?

Consider the whole set of options including the repl flag:

```bash
node -e "var myVar = './stuff'; console.log(myVar);"
```